pkg/cmd/registry/rule/rulecmdutil/flagset.go

```diff
@@ -0,0 +1,107 @@
+package rulecmdutil
+
+import (
+	"github.com/redhat-developer/app-services-cli/pkg/cmd/registry/registrycmdutil"
+	"github.com/redhat-developer/app-services-cli/pkg/core/cmdutil/flagutil"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/factory"
+	"github.com/spf13/cobra"
+)
+
+type flagSet struct {
+	cmd     *cobra.Command
+	factory *factory.Factory
+	*registrycmdutil.FlagSet
+}
+
+// NewFlagSet returns a new flag set with common Service Registry rule flags
+func NewFlagSet(cmd *cobra.Command, f *factory.Factory) *flagSet {
+	return &flagSet{
+		cmd:     cmd,
+		factory: f,
+		FlagSet: registrycmdutil.NewFlagSet(cmd, f),
+	}
+}
+
+// AddArtifactID adds a flag for setting the artifact ID
+func (fs *flagSet) AddArtifactID(artifactID *string) {
```

Comment:
@wtrocki thinking to move it to `registrycmdutil`so we can have it for artifacts cmd too.